res/css/views/rooms/_EventTile.pcss

```diff
@@ -270,7 +270,11 @@ $left-gutter: 64px;
     }
 
     &[data-layout="irc"] {
-        --EventTile_irc_line_info-margin-inline-start: calc(var(--name-width) + 10px + var(--icon-width));
+        /* add --right-padding value of MessageTimestamp only */
+        /* stylelint-disable-next-line declaration-colon-space-after */
+        --EventTile_irc_line_info-margin-inline-start: calc(
+            var(--name-width) + var(--icon-width) + 1 * var(--right-padding)
```

Comment:
`1 *` is set to clarify that right-padding value of just one element (MessageTimeStamp) is required to be added.